### PR TITLE
Setup demo-app with webpack-react.

### PR DESCRIPTION
**Pull request by pallab-gain**
https://app.clubhouse.io/callstatsio/story/7761/setup-demo-app-with-webpack-react

**Setup demo-app with webpack-react.**
Replace the existing folder structure with webpack boilerplate taken from https://github.com/antonfisher/react-express-webpack . Also removed the redux, material UI parts for simplicity right now. To make sure whether we can progress with this stack the landing page UI for demo application is also included here as a test. Story link for landing page UI - [CH7750]( https://app.clubhouse.io/callstatsio/story/7750/landing-page-ui-for-demo-app)

_There is no logic attached to the UI right now. Also the readme, settings and other components are omitted, and will be added with logic implementation story._

### CODE REVIEW

**Comment by pallab-gain** [comment]
Thanks for the suggestions. I agree that master branch should always be deploy-able. Hence, I was considering ch7461 as the feature branch, and considered it the base for this pull request. Thanks for helping me with the naming hint. I will renamed this ch7461 to more meaningful one based on the suggested.

File: src/client/components/applayout/AppLayout.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import HeaderLayout from './HeaderLayour';
+import FooterLayout from './FooterLayout';
+import ContentLayout from './ContentLayout';
+export class AppLayout extends React.Component {
+  render() {
+    return (
+      <div className={'container-fluid'}>
```
Comment (juhocallstats) [inline src/client/components/applayout/AppLayout.js]: String props can be passed directly without brackets/expressions `<div className='container-fluid'>`

File: src/client/components/applayout/ContentLayout.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+class CreateMeetingLayout extends React.Component {
+  render() {
+    const customStyle = {
```
Comment (juhocallstats) [inline src/client/components/applayout/ContentLayout.js]: Check out styled components for more easy styling https://www.styled-components.com/docs/basics#getting-started

File: src/client/components/applayout/ContentLayout.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+              Creating meeting</button>
+            </li>
+          </ul>
+        </div>
+      </div>
+    );
+  }
+}
+
+class OtherComponentLayout extends React.Component {
```
Comment (juhocallstats) [inline src/client/components/applayout/ContentLayout.js]: It's more clear if there's only 1 component per file, which would then be the default export

**Pull request merged**